src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2315,6 +2377,97 @@
             "required": [ "name" ]
         },
 
+        "toolComponentTranslation": {
+            "description": "TBD.",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "toolComponentGuid": {
+                    "description": "TBD must be GUID format: matches toolComponent.guid.",
+                    "type": "string"
+                },
+
+                "location": {
+                    "description": "The location where translation can be obtained.",
+                    "$ref": "#/definitions/artifactLocation"
+                },
+
+                "semanticVersion": {
+                    "description": "The semantic version of the tool component for which the translation was made.",
+                    "type": "string"
+                },
+
+                "partialTranslation": {
+                    "description": "Should be true if this object contains a subset of the strings defined by the tool component.",
```

Comment:
>Should be  [](start = 36, length = 10)

Omit "Should be".

See how impossible I am? In this case, I _do_ like my exact text in the eballot proposal comment" :-)